src/main/java/org/prebid/server/validation/RequestValidator.java

```diff
@@ -508,13 +564,50 @@ private void validateNativeVideoProtocols(List<Integer> protocols, int impIndex,
 
     private void validateNativeVideoProtocol(Integer protocol, int impIndex, int assetIndex, int protocolIndex)
             throws ValidationException {
-        if (protocol < 0 || protocol > 10) {
+        if (protocol < Protocol.VAST10.getValue() || protocol > Protocol.DAAST10_WRAPPER.getValue()) {
             throw new ValidationException(
                     "request.imp[%d].native.request.assets[%d].video.protocols[%d] must be in the range [1, 10]."
                             + " Got %d", impIndex, assetIndex, protocolIndex, protocol);
         }
     }
 
+    private void validateNativeEventTrackers(List<EventTracker> eventTrackers, int impIndex)
+            throws ValidationException {
+
+        if (CollectionUtils.isNotEmpty(eventTrackers)) {
+            for (int eventTrackerIndex = 0; eventTrackerIndex < eventTrackers.size(); eventTrackerIndex++) {
+                validateNativeEventTracker(eventTrackers.get(eventTrackerIndex), impIndex, eventTrackerIndex);
+            }
+        }
+    }
+
+    private void validateNativeEventTracker(EventTracker eventTracker, int impIndex, int eventIndex)
+            throws ValidationException {
+        if (eventTracker != null && (eventTracker.getEvent() < EventType.IMPRESSION.getValue()
```

Comment:
Please extract `eventTracker.getEvent()` result to a local variable and mind possible `null`